src/raftstore/store/util.rs

```diff
@@ -524,6 +524,12 @@ impl Lease {
         }
     }
 
+    /// Returns the current `Timespec`.
+    #[inline]
+    pub fn now() -> Timespec {
```

Comment:
It looks weird. Is there any examples that use associated function to create an instance of different type?